deepchem/data/data_loader.py

```diff
@@ -979,7 +979,7 @@ def shard_generator():  # TODO Enable sharding with shard size parameter
         # (X, y, w, ids)
         yield X, None, None, ids
 
-    def _read_file(input_file: str, auto_add_annotations: bool = False):
+    def _read_file(input_file: str):
```

Comment:
Was this a bug in the earlier PRs that you're fixing?

Reply:
Ah, yes, must not have cleaned this branch properly.

Should this stay in the PR, or should I take it out (and we can merge the separate bug fix PR)?

Reply:
I think it's fine to have this change stay in the PR